gbpservice/nfp/configurator/agents/loadbalancer_v1.py

```diff
@@ -0,0 +1,655 @@
+import os
+import oslo_messaging as messaging
+
+from gbpservice.nfp.configurator.agents import agent_base
+from oslo_log import log as logging
+from gbpservice.nfp.core.main import Event
+from gbpservice.nfp.configurator.lib import utils
+from gbpservice.nfp.configurator.lib.filter import Filter
+from gbpservice.nfp.core import poll as nfp_poll
+from gbpservice.nfp.configurator.lib import lb_constants
+
+LOG = logging.getLogger(__name__)
+
+
+class LBaasRpcSender(Filter):
+    """Agent side of the Agent to Plugin RPC API."""
```

Comment:
Please follow the class commenting style as done in configurator as that was followed by all.